### ISSUE TITLE
Minor redesign of options page

### ISSUE THREAD

**Issue by dhowe** [vtoubiana/TrackMeNot#90]
for a) clarity, b) information architecture, and c) aesthetics:

- [ ] grouping of options should make more sense, also order by priority (important at top)
- [ ] logging: disabled should grey-out other options
- [ ] aesthetics: alignment, line-breaking, spacing, etc.
- [ ] buttons shouldn't move when depressed
- [ ] identify mouse-over links to FAQ
- [ ] credits at bottom?

**Comment by cqx931** [comment]
**Icon**
- Remove queries and error from Icon
- Showing stealth(dark blue)/tab(orange)/off(red) status?

**Menu**
- Enable / Disable
- Stealth/Tab
  - Disable stealth/tab option if disabled
  - Change the text to "Switch to Stealth/Tab mode" ?
- Options
- Help

**Options Page**
-  (Remove enable/disable, stealth/tab options from options page)
- Logging  [checkbox, default:enabled] 
  - 'Show logs/Hide logs'[button]
  -  normal [radio button, defulat]
  -  Persistent mode [radio button]
  -  (Move 'Clear logs' button to the log interface)
- Burst Mode [checkbox, default disabled]
- Avg Quary Rate [Selection]

- Search Engine
  - Remove the text 'Selection'

- Queries
  - 'Show Queries/Hide Queries' [button]
  - RSS Feed
  - Black List (remove the ‘Use List’ checkbox)
  - Generate queries including keywords monitored by DHS [checkbox, default: disabled]

- Apply Button

- Links
  - Main Site
  - Help/FAQ
- Credits

**Comment by dhowe** [comment]
Perhaps we should show the number of queries sent on the icon badge? So that users can see progress...

The order above should be:

Search Engines
Avg Query Rate / Burst mode
Queries
Logging
Links
Credits

Do we need an apply button? Or can settings take effect immediately?

Main Site [Link] = optional

**Comment by cqx931** [comment]
> Perhaps we should show the number of queries sent on the icon badge? So that users can see progress...
Not sure about this... The number of queries can go really large after a long time and it's not a good place to show large number. How about showing some progress dots whenever a new query is being sent? Similar to the idea of 'ad clicking' in AdNauseam.

Agree - no need to add the apply button.

**Comment by dhowe** [comment]
We could change the number when it gets over 999, to something ">1k", ">10k", ">100k", or we could show the # of queries since the last restart. But I like the idea of the user being able to see their progress... @vtoubiana, any thoughts here?

**Comment by cqx931** [comment]
![image](https://user-images.githubusercontent.com/3056154/27693034-d5e3cbf6-5d1a-11e7-9499-a5a8e7a934d0.png)

**Comment by cqx931** [comment]
Another option is merging "Burst" into "Quiries", after Query Frequency.

**Comment by dhowe** [comment]
Yes, I was going to suggest that; perhaps to the right of frequency add 'enable bursts [-]', with a quesiton-mark link to FAQ ?

**Comment by cqx931** [comment]
![image](https://user-images.githubusercontent.com/3056154/27735844-1da42846-5dd4-11e7-9b74-277deeb9a5d9.png)

**Comment by dhowe** [comment]
Great (thous should be 'Enable Bursts', plural.
But we also need question marks for others:

- logging options
- generate dhs queries 
- rss 
- blacklist

**Comment by cqx931** [comment]
Changes pushed to a new branch: https://github.com/vtoubiana/TrackMeNot/tree/optionPage

**Comment by greysonevins** [comment]
- [ ] Persistent Logging
- [ ] Save/Refactor Logging Function
